scripts/pre_commit_linter.py

```diff
@@ -179,6 +227,7 @@ def _lint_js_files(node_path, jscs_path, config_jscsrc, files_to_lint):
             num_files_with_errors += 1
             print linter_stdout
 
+    print 'Linting complete'
```

Comment:
I don't think this is needed; it suggests all the linting is done when the Python linting hasn't started yet. It can be omitted, I think.